components/social-media/AuthWithSocialMedia.tsx

```diff
@@ -1,32 +1,20 @@
 import clsx from 'clsx'
-import { Image, Pressable, Text, View } from 'react-native'
+import { ReactNode } from 'react'
+import { View } from 'react-native'
 
-const imageGoogleIcon = require('../../assets/images/google.png')
-const imageGithubIcon = require('../../assets/images/github.png')
-const imageTwitterIcon = require('../../assets/images/twitter.png')
+interface Props {
+  position: boolean
+  children: ReactNode
```

Comment:
eliminar importacion del tipo ReactNode para la prop children, en su lugar : 
```suggestion
  children: React.ReactNode
```